cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -1106,6 +1107,11 @@
 
     // The guice binding name for http handler used by the messaging system
     public static final String HANDLER_BINDING_NAME = "messaging.http.handler";
+
+    public static final String HBASE_METADATA_TABLE_NAME = "cdap.messaging.metadata.table.name";
+    public static final String HBASE_METADATA_TABLE_NAMESPACE = "cdap.messaging.metadata.hbase.namespace";
+
+    public static final String COPROCESSOR_DIR = "messaging.coprocessor.dir";
```

Comment:
Can you move this up together with other cConf property keys? Also add a comment for the `HBASE_METADATA_TABLE_*` constants, as they are not cConf keys, but rather HBase table attributes (hence need the `cdap.` prefix, I believe).